installer_scripts/install-yb-voyager

```diff
@@ -75,6 +77,19 @@ main() {
 			centos_main
 			;;
 		"AlmaLinux")
+			sudo yum -qy module disable postgresql
+			centos_main
+			;;
+		"Red Hat Enterprise Linux")
+			version=$(cat /etc/os-release | grep -w VERSION_ID | awk -F'=' '{print $2}' | tr -d '"' | awk -F'.' '{print $1}')
+			if [ $version -ge 8 ] 
+			then
+				#disable default PostgreSQL repo
+				sudo yum -qy module disable postgresql
+			fi
```

Comment:
Define a new function named `disable_default_pg_repo()` and move all this code in that function.
Call `disable_default_pg_repo` for both Alma and RHEL cases.

Reply:
Do I put this function in the "COMMON" section or the "CENTOS" section

Reply:
Guessing from the function prototypes I have put it in the "COMMON" section of the script

Reply:
You can put it in the centos section.

Reply:
Done

Reply:
I'm sorry I missed your response on the above thread about not requiring the version check :| 
Alma Linux is only version 8 and above. So technically it should not require any version check as well.
And finally do I remove the version check completely from the newly added function and just run the disable command? Let me know.

Reply:
Yes. Remove the version check. Just run the command. In that case, we don't even need a function. :-)

Reply:
Change done and verified on AlmaLinux and RHEL machine as well. Will remove the machines once you review and pass the changes :P